libs/lb/lb-rs/src/service/network.rs

```diff
@@ -68,7 +68,13 @@ impl Network {
             client_version: client_version.clone(),
         })
         .map_err(|err| ApiError::Serialize(err.to_string()))?;
+
+        if serialized_request.len() > 1024 * 1024 {
+            tracing::warn!("making network request with {} bytes", serialized_request.len());
+        }
```

Comment:
I think canvas users will see this for practically every `svg`. Maybe 10mb is a more suitable threshold

Reply:
also maybe you want 10mb in plaintext somewhere for ease